packages/styles/forms.css

```diff
@@ -267,6 +270,16 @@ textarea.Field--has-error:focus:hover,
   border: 1px solid var(--field-border-color);
 }
 
+.Field__labelDescription {
+  flex-basis: 100%;
+  color: var(--field-label-description-text-color);
+  font-size: var(--text-size-small);
+  font-weight: var(--font-weight-normal);
+  line-height: 1;
+  margin: 0 0 0 30px;
```

Comment:
Nitpick: This should probably match the computed width of the icon, unfortunately we don't have icon size anywhere as a variable so that is something that we should probably refactor in cauldron somewhere else. At a minimum, we could use calc:

```css
margin-right: calc(24px + 2px + var(--space-half));
```

* `24px` - icon size
* `2px` - borders (left and right)
* `var(--space-half)` matching the natural spacing between the input and the text

I don't recall why we put an invisible border around the input though 🤔

Reply:
fair enough, i was looking for an icon size var and didn't see one. I will update to use the proposed value.

Reply:
> Might want to pass this by @awpearlm as well if he's not already aware

@awpearlm - Had a CSS change for the desc text that @bobbyomari will be working on. Once he has a design, he will link it to the original issue #513 and I will make the update. Aaron also was interested in adding this prop to the checkbox element as well. I will create a new issue for that in GH.

Reply:
Reviewed the changes with @awpearlm, he approves.